src/main/resources/lang/en.xml

```diff
@@ -366,4 +362,19 @@
         <snowflake>snowflake</snowflake>
         <snowflakes>snowflakes</snowflakes>
     </snowflakes>
+    <death>
```

Comment:
Don't worry about ocn's paths here. Make a conversion method to translate if you need to. We have a whole new system coming for cardinal2 when that gets out the door.